Create audio target selector for OSC/Superdirt

daslyfe opened the pull request:
Allows you to set the default target to osc so you do not have to write .osc() after everything if you are using superdirt as an audio backend. (webaudio is the default of course) Also: 
- fixes a bug in webkit browsers where confirm would happen automatically regardless of user choice 
- fixes OSC jitter for the web version of strudel
<img width="593" alt="Screenshot 2024-08-11 at 12 40 27 PM" src="https://github.com/user-attachments/assets/587e67cf-5d4c-4365-9cc9-68d4facbb252">

yaxu commented:
This looks good! I guess someone might select this by mistake, and then puzzle over why strudel stops working.. But they could always click the 'restore default settings' button in that case.

daslyfe commented:
> This looks good! I guess someone might select this by mistake, and then puzzle over why strudel stops working.. But they could always click the 'restore default settings' button in that case.

I added 
<img width="518" alt="Screenshot 2024-08-31 at 11 35 18 PM" src="https://github.com/user-attachments/assets/7aa1c14c-1a9c-4f2f-8084-6b75d9492b02">
a warning message that points to the docs, and updated the docs as well :)

daslyfe commented:
I plan on merging this tomorrow morning after extensive testing
